### ISSUE TITLE
Country Unknown

### ISSUE THREAD

**Issue by m-primo** [aptabase/aptabase#58]
Country always unknown, I have tested multiple countries using real IPs and VPN IPs and still unknown.

![image](https://github.com/aptabase/aptabase/assets/44984918/2b0221b9-5ac7-46cd-9af7-800b82065a81)

P.S.: I think it has something to do with reverse proxy, I do have reverse proxy using Cloudflare DNS and Apache.
It's exactly like this:
Cloudflare -> Apache -> aptabase docker container

**Comment by goenning** [comment]
We use the client IP to determine country/region, but because it’s behind 2 reverse proxy, you’ll need to forward it from CF to Apache and then from Apache to Docker.

It shows as unknown because docker is probably seeing 127.0.0.1 as the client ip (Apache)

**Comment by m-primo** [comment]
> We use the client IP to determine country/region, but because it’s behind 2 reverse proxy, you’ll need to forward it from CF to Apache and then from Apache to Docker.
> 
> It shows as unknown because docker is probably seeing 127.0.0.1 as the client ip (Apache)

I suspected that at first, but I wasn't really sure about it.
I was sure about Cloudflare to Apache, because I had other hosts/services and the IP was correct, but I wasn't sure about Apache to Docker, anyway I have replaced Apache with Nginx and made sure that X-Real-IP and some other related headers, and now it's working properly. Thanks, and sorry for any inconvenience.

I have another unrelated question if I may ask, How do I can disable registration, I have looked everywhere but I couldn't find anything?
